src/applications/mhv-secure-messaging/containers/ThreadDetails.jsx

```diff
@@ -120,7 +123,11 @@ const ThreadDetails = props => {
       return (
         <>
           <va-loading-indicator
-            message="Sending message..."
+            message={
+              largeAttachmentsEnabled
+                ? 'Do not refresh the page. Sending message...'
+                : 'Sending message...'
+            }
```

Comment:
these two options for the message are in three different places.  Could they become constants?